olp-cpp-sdk-core/src/cache/DefaultCacheImpl.cpp

```diff
@@ -843,6 +852,16 @@ DefaultCache::StorageOpenResult DefaultCacheImpl::SetupProtectedCache() {
     return ToStorageOpenResult(status);
   }
 
+  if (!protected_cache_->CheckState()) {
+    OLP_SDK_LOG_ERROR_F(kLogTag,
```

Comment:
This should rather be clear. E.g. "Protected cache open failed, L0 files > 4, opening DefaultCache without protected cache, path='%s'"

Reply:
This is removed because it is not needed after revert of #1217.